### PR TITLE
Keep coordinate system data

### PR DESCRIPTION
**Pull request by regisoc**
This PR adds a tracking of coordinate systems related to electrodes from BIDS archives.
Requires a DB update from Loris Core [PR 8242](https://github.com/aces/Loris/pull/8242).

### CODE REVIEW

File: python/bids_import.py [hunk 1]
```diff
@@ -271,8 +271,8 @@ def create_loris_bids_directory(bids_reader, data_dir, verbose):
 
     # determine the root directory of the LORIS BIDS and create it if does
     # not exist
-    name = re.sub("[^0-9a-zA-Z]+", "_", bids_reader.dataset_name) # get name of the dataset
-    version = re.sub("[^0-9a-zA-Z\.]+", "_", bids_reader.bids_version) # get BIDSVersion of the dataset
+    name = re.sub("[^0-9a-zA-Z]+", "_", bids_reader.dataset_name)        # get name of the dataset
+    version = re.sub("[^0-9a-zA-Z\\.]+", "_", bids_reader.bids_version)  # get BIDSVersion of the dataset
```
Comment (laemtl) [inline python/bids_import.py:275]: Est0ce que ça fonctionne encore si on escape le slash?

Comment (regisoc) [reply python/bids_import.py]: The old version triggered a [Flake W605](https://www.flake8rules.com/rules/W605.html) on my end, but that was not a proper solution. I updated it, it should be good now with `r"regex"` string pattern.

**Comment by cmadjar** [comment]
@regisoc please rebase to 24.1-release branch

**Comment by regisoc** [comment]
After discussion, no rebase needed.

**Comment by regisoc** [comment]
Blocked #882 - need subproject -> cohort name changing

**Comment by regisoc** [comment]
Rebased #885

**Pull request closed without merging**